src/input_xml.F90

```diff
@@ -97,6 +103,17 @@ subroutine read_settings_xml()
     ! Parse settings.xml file
     call open_xmldoc(doc, filename)
 
+    ! Find if a multi-group or continuous-energy simulation is desired
+    if (check_for_node(doc, "energy_mode")) then
+      call get_node_value(doc, "energy_mode", temp_str)
+      temp_str = trim(to_lower(temp_str))
+      if (temp_str == "mg" .or. temp_str == "multi-group") then
+        run_CE = .false.
+      else if (temp_str == "ce" .or. temp_str == "continuous") then
```

Comment:
The documentation says "continuous-energy" rather than "continuous". Pick one and be consistent.